ephios/api/urls.py

```diff
@@ -14,12 +14,23 @@
     ApplicationDetail,
     ApplicationUpdate,
 )
-from ephios.api.views.events import EventViewSet
-from ephios.api.views.users import UserProfileMeView
+from ephios.api.views.events import EventViewSet, ShiftViewSet
+from ephios.api.views.users import (
+    UserByMailView,
+    UserParticipationView,
+    UserProfileMeView,
+    UserViewSet,
+)
 from ephios.extra.permissions import staff_required
 
 router = routers.DefaultRouter()
 router.register(r"events", EventViewSet)
+router.register(r"shifts", ShiftViewSet)
+router.register(r"users", UserViewSet)
+router.register(r"users/by_email", UserByMailView, basename="user-by-email")
+router.register(
+    r"users/(?P<user>[\d]+)/participations", UserParticipationView, basename="user-participations"
```

Comment:
We should add a trailing `/` everywhere ;)

Reply:
you sure? these are only the prefixes for the viewset and DRF appends additonal URL parts to construct the actual endpoints

Reply:
Ah well true nvmd